docs/_getting-started-macos-android.md

```diff
@@ -26,7 +26,7 @@ brew tap homebrew/cask-versions
 brew install --cask zulu11
 ```
 
-The Zulu OpenJDK distribution offers JDKs for **both Intel and M1 Macs**. This will make sure your build are faster on M1 Macs compared to using an Intel-based JDK.
+The Zulu OpenJDK distribution offers JDKs for **both Intel and M1 Macs**. This will make sure your builds faster on M1 Macs compared to using an Intel-based JDK.
```

Comment:
Isn't this a typo now?
It should be eihter:
- This will make sure your builds are faster
- This will make sure you build faster

Reply:
Yup, good catch, it was typo before, and it is new typo now, let me update it. 😅